testing/TestHarness/TestBackend/Controllers/CustomAuthController.cs

```diff
@@ -62,7 +63,7 @@ public void RefreshCookie()
 					HttpOnly = true,
 					SameSite = SameSiteMode.None
 				});
-				Response.Cookies.Append("RefreshToken", $"Refresh-{DateTime.Now.ToString("HH:mm:sss")}" + token, new CookieOptions
+				Response.Cookies.Append("RefreshToken", $"Refresh-{DateTime.Now.ToString("HH:mm:sss")}|" + token, new CookieOptions
```

Comment:
Should this use invariant culture?

Reply:
possibly, but it's only for the test harness